test/negative/spirv_report_bad_input.spt

```diff
@@ -0,0 +1,32 @@
+; RUN: llvm-spirv %s -to-binary -o %t.spv
+; RUN: echo "0" > %t_corrupted.spv && cat %t.spv >> %t_corrupted.spv
+; RUN: not llvm-spirv --spirv-print-report %t_corrupted.spv 2>&1 | FileCheck %s --check-prefix=CHECK-ERROR
+;
+; CHECK-ERROR: Invalid SPIR-V binary
+
```

Comment:
Perhaps add a comment explaining what's bad about this input.

Reply:
I'll add a comment to the test file explaining what's bad about this input.

However, if talking about possible output from `llvm-spirv --spirv-print-report`, more user friendly output will be available later, after https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2304 will add a call to get human-readable info by an error code (std::string getErrorMessage(int ErrCode);) This test file will then serve to check both #2298 and #2304

Reply:
@asudarsa Please see also the above comment wrt. your comment about testing of https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/2304